src/sql/workbench/services/notebook/browser/notebookService.ts

```diff
@@ -128,9 +128,21 @@ export interface IProviderInfo {
 	providerId: string;
 	providers: string[];
 }
+
+export interface INotebookInput {
```

Comment:
I needed to abstract this so that the notebook service wasn't directly refering to the input. This should be refactored to invert this dependency (I think this is mainly cause by the angular dependency in notebooks), but that is future work.